src/components/header/modals/ModalNetwork.vue

```diff
@@ -48,13 +48,15 @@
                 </div>
               </label>
               <!-- custom endpoint -->
-              <input
-                v-if="provider.key === endpointKey.CUSTOM && selNetwork === endpointKey.CUSTOM"
-                v-model="newEndpoint"
-                type="text"
-                placeholder="IP Address / Domain"
-                class="ip-input"
-              />
+              <div class="row--ip-input">
```

Comment:
![Screen Shot 2022-04-04 at 11 22 21 AM](https://user-images.githubusercontent.com/40356749/161514454-aef9cb2a-f49d-4838-9d55-c8ba0bc3f6b8.png)

The width of the network button (`314px`) and the width of the network input field (`330px`) seems to be different.
@Kahonnohak is this intentional?

Reply:
@hoonsubin @sirius651
I think [this](https://gyazo.com/219e1478b04b387ef9884fa3668be4b1) is the original design made by @Kahonnohak 
Let me fix the styling quickly.

Reply:
Fixed.

![Screenshot 2022-04-04 at 6 40 11 PM](https://user-images.githubusercontent.com/92044428/161527918-303ee482-cc0c-4630-9b15-4d39e249fa4f.png)

![Screenshot 2022-04-04 at 6 39 59 PM](https://user-images.githubusercontent.com/92044428/161527913-bb1d5e6d-0c8c-45ba-a044-1ab580dbb4df.png)

![Screenshot 2022-04-04 at 6 39 43 PM](https://user-images.githubusercontent.com/92044428/161527906-cfee9699-6e7c-4abc-a1ea-518b21350ced.png)